common/analysis/lint_rule_status.cc

```diff
@@ -58,6 +58,39 @@ void LintStatusFormatter::FormatLintRuleStatus(std::ostream* stream,
   }
 }
 
+void LintStatusFormatter::FormatLintRuleStatuses(
```

Comment:
Recommend factoring this into two functions, one that sort, the other that format the sorted result.

Reply:
No longer relevant since switching to sets